mix.exs

```diff
@@ -41,7 +50,10 @@ defmodule Litmus.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:ex_doc, "~> 0.18.0", only: :dev, runtime: false}
+      {:ex_doc, "~> 0.18.0", only: :dev, runtime: false},
+      {:credo, "~> 0.9.1", only: [:dev, :test], runtime: false},
+      {:excoveralls, "~> 0.8", only: :test},
+      {:dialyxir, "~> 0.5", only: [:dev], runtime: false}
```

Comment:
we can remove the `[]` here, to be consistent with the other dependencies.

Reply:
added test mode to dialyxir